src/transformers/modeling_tf_ctrl.py

```diff
@@ -121,9 +120,6 @@ def call(self, inputs, training=False):
             k = tf.concat((past_key, k), axis=-2)
             v = tf.concat((past_value, v), axis=-2)
 
-        # to cope with keras serialization
-        use_cache = cast_bool_to_primitive(use_cache, True)
-
         if use_cache is True:
```

Comment:
Be careful with these `is True` statements. Since they won't work if `True` is accidentally cast to `tf.constant(True)`.

Reply:
Yes! I'm aware of this. The cast to primitive is used only for this use_cache parameter because it cannot be handled properly otherwise.